### PR TITLE
Added latex info to README.rst. 

### LINKED ISSUE #154: List of required latex packages?
I've got a lot on my system already, so can't give a full list, but had to install

- latexmk
- fncychap
- tabulary 
- tocloft

to get the pdf docs to build

### PR DESCRIPTION
**Pull request by MichaelClerx**
Closes #154.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -50,14 +50,26 @@ The built documentation will be available at::
...
 
   make latexpdf
 
-**Note:** creating the PDF form of the documentation requires that some or all of a tool like Windows/`MikTeX <https://miktex.org/>`_, Windows/`TeXlive <https://www.tug.org/texlive/>`_, Linux/`TeXlive <https://www.tug.org/texlive/>`_, and macOS/`MacTeX <https://tug.org/mactex/>`_ is available to the Sphinx build tool.
-Also check that you have ``latexmk`` available as this is required for creating the PDF form of the documentation.
+If this fails, please have a look at the following:
+
+Creating the PDF form of the documentation requires that some or all of a tool like Windows/`MikTeX <https://miktex.org/>`_, Windows/`TeXlive <https://www.tug.org/texlive/>`_, Linux/`TeXlive <https://www.tug.org/texlive/>`_, and macOS/`MacTeX <https://tug.org/mactex/>`_ is available to the Sphinx build tool.
+
+In addition, you'll need the following latex packages:
```
Comment (agarny) [inline README.rst:64]: "LaTeX", not "latex".

File: README.rst [hunk 2]
```diff
@@ -50,14 +50,26 @@ The built documentation will be available at::
...
+
+`babel-greek`, `capt-of`, `cbfonts`, `courier`, `fncychap`, `fontspec`, `framed`, `greek-fontenc`, `helvetic`, `latexmk`, `needspace`, `tabulary`, `textgreek`, `titlesec`, `tocloft`, `varwidth`, `wrapfig`
```
Comment (agarny) [inline README.rst:66]: Is the intention to render this as code style? If so, you need to use double back ticks.

Comment (agarny) [reply README.rst:66]: Also, I would put that list directly after the colon and end it with a full stop.

File: README.rst [hunk 3]
```diff
@@ -50,14 +50,26 @@ The built documentation will be available at::
...
+
+In particular, check that you have ``latexmk`` available as this is required for creating the PDF form of the documentation.
 Information on ``latexmk`` can be found at https://ctan.org/pkg/latexmk/.
 
+You may also need to install the ["GNU FreeFont" fonts](https://www.gnu.org/software/freefont/) `FreeSans`, `FreeSerif`, `FreeMono`.
```
Comment (agarny) [inline README.rst:71]: Double back ticks again. Also, add "and" before `FreeMono`.

File: README.rst (hunk 1 above)
Comment (MichaelClerx) [reply README.rst:64]: That's marketing!

File: README.rst (hunk 2 above)
Comment (MichaelClerx) [reply README.rst:66]: Thanks!

File: README.rst (hunk 3 above)
Comment (MichaelClerx) [reply README.rst]: Thanks, just fixed this!

File: README.rst (hunk 1 above)
Comment (agarny) [reply README.rst]: Having seen "LaTeX" all those years, "latex" looks really odd.

File: README.rst [hunk 4]
```diff
@@ -61,14 +61,12 @@ If this fails, please have a look at the following:
 
 Creating the PDF form of the documentation requires that some or all of a tool like Windows/`MikTeX <https://miktex.org/>`_, Windows/`TeXlive <https://www.tug.org/texlive/>`_, Linux/`TeXlive <https://www.tug.org/texlive/>`_, and macOS/`MacTeX <https://tug.org/mactex/>`_ is available to the Sphinx build tool.
 
-In addition, you'll need the following latex packages:
-
-`babel-greek`, `capt-of`, `cbfonts`, `courier`, `fncychap`, `fontspec`, `framed`, `greek-fontenc`, `helvetic`, `latexmk`, `needspace`, `tabulary`, `textgreek`, `titlesec`, `tocloft`, `varwidth`, `wrapfig`
+In addition, you'll need the following latex packages: ``babel-greek``, ``capt-of``, ``cbfonts``, ``courier``, ``fncychap``, ``fontspec``, ``framed``, ``greek-fontenc``, ``helvetic``, ``latexmk``, ``needspace``, ``tabulary``, ``textgreek``, ``titlesec``, ``tocloft``, ``varwidth``, ``wrapfig``.
```
Comment (agarny) [inline README.rst:64]: Add "and" before `wrapfig`.

File: README.rst (hunk 1 above)
Comment (MichaelClerx) [reply README.rst]: Ok fixed it for you

File: README.rst (hunk 3 above)
Comment (agarny) [reply README.rst]: ?? Where?

File: README.rst (hunk 1 above)
Comment (agarny) [reply README.rst]: Woo hoo! Thanks @MichaelClerx. :)

File: README.rst (hunk 3 above)
Comment (agarny) [reply README.rst]: Didn't ou intend `FreeSans`, `FreeSerif` and `FreeMono` to be in code style? (Also add "and" before `FreeMono`.)

**Pull request merged**